roads.mss

```diff
@@ -2579,9 +2579,14 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
 }
 
 #guideways {
+  [zoom >= 11][zoom < 13] {
```

Comment:
I guess `[zoom < 13] ` is not needed, because next condition will just take over.

Reply:
It's actually just a cut and paste effect (I got that line of code from the railway code where "low zoom" is handled separately), but yo could also call it defensive coding :)

Interestingly, less mapnik.xml is created with the extra test, so I'm inclined to leave it in:

with:
wc mapnik.xml
45390  161881 2617683 mapnik.xml

without:
wc mapnik.xml
45396  161894 2618000 mapnik.xml